bin/fetch-cobolcheck.ps1

```diff
@@ -23,5 +23,5 @@ Function Get-DownloadUrl {
 }
 
 $downloadUrl = Get-DownloadUrl
-$outputFile = Join-Path -Path $PSScriptRoot -ChildPath "cobolcheck.exe"
+$outputFile = Join-Path -Path "${HOME}" "cobolcheck" "cobolcheck.exe"
```

Comment:
Does `Invoke-WebRequest` create the `cobolcheck` directory when it doesn't yet exist?

Reply:
I guess, no. Didn't test it. I added additional `New-Item` command to create it now.